src/frontend/src/lib/components/carousel/Carousel.svelte

```diff
@@ -114,6 +116,9 @@
 	 * Start autoplay timer
 	 */
 	const initialiseAutoplayTimer = () => {
+		
+		if (CAROUSEL_AUTOPLAY_DISABLED) {return}
```

Comment:
I'm not that a fan of adding any logic related to E2E tests within the actual dApp code. Kind of consider this a bad practice, as it can potentially lead to issues in production.

Have you considered alternative solutions?

Reply:
I did try to solve it differently (tell playwright to disable animations, use css injections to disable transitions) but nothing worked out the way it should. 

Maybe you have an idea?

Reply:
I think that one of the alternatives was to manipulate CSS/DOM to stop the carousel, or at least to remove the slides (except one).

Talking with @peterpeterparker , it seems that `playwright` offers something for that

```typescript
  await page.evaluate(() => {
    const elements = document.querySelectorAll('SlidesExceptOne')
    elements.forEach(element => element.remove());
  });
```

Reply:
Have you try to remove elements (the slides expect first) in the DOM?

Reply:
Feedback discussed offline and re-worked.